static/features.html

```diff
@@ -339,12 +338,12 @@ <h4><a href="#exploit-mitigations">Exploit mitigations</a></h4>
                                     including many which we played a part in developing and
                                     landing upstream as part of our linux-hardened project (which
                                     we intend to revive as a more active project again).</li>
-                                    <li>Forced kernel module signing with per-build RSA 4096 /
-                                    SHA256 keys and lockdown mode set to forced confidentiality
+                                    <li>Forced kernel module signing with per-build RSA 4096/SHA256
```

Comment:
The space in this slash is intentional because these are RSA 4096 SHA 256 certificates (SHA256withRSA), this needs to be dropped too.